consensus/src/quorum/verifiers.rs

```diff
@@ -131,18 +131,10 @@ pub fn verify_votes(
 }
 
 impl Cluster<PublicKey> {
-    fn aggregate_pks(&self) -> Result<BlsAggPublicKey, StepSigError> {
+    fn aggregate_pks(&self) -> Result<BlsMultisigPublicKey, StepSigError> {
         let pks: Vec<_> =
             self.iter().map(|(pubkey, _)| *pubkey.inner()).collect();
-
-        match pks.split_first() {
-            Some((first, rest)) => {
-                let mut apk = BlsAggPublicKey::from(first);
-                apk.aggregate(rest)?;
-                Ok(apk)
-            }
-            None => Err(StepSigError::EmptyApk),
-        }
+        Ok(BlsMultisigPublicKey::aggregate(&pks)?)
```

Comment:
Nice, I really like this API change. Didn't check the [PR on BLS-sign](https://github.com/dusk-network/bls12_381-bls/pull/19) before, but nice.